enumerable.rb

```diff
@@ -0,0 +1,15 @@
+require './my_enumerable'
```

Comment:
- [ ] Although using require is ok and its working but using `require_relative` is ruby convention for files in your directory so I will recommend that you use require_relative here. `require` is used for external files.